certbot-compatibility-test/nginx/nginx-roundtrip-testdata/piwik/nginx.conf

```diff
@@ -22,9 +22,9 @@ server {
     root  /var/www/sites/stats.example.com/;
     index  index.php index.html;
 
-    # Disallow any usage of piwik assets if referer is non valid.
+    # Disallow any usage of piwik assets if referrer is non valid.
     location ~* ^.+\.(?:jpg|png|css|gif|jpeg|js|swf)$ {
-             # Defining the valid referers.
+             # Defining the valid referrers.
```

Comment:
These typos should not be fixed, because this file is copied verbatim from elsewhere.

Reply:
Done.